uwblueprint/ccbc
# responsive homepage functionality

## Notion ticket link

[Responsive Home Page](https://www.notion.so/uwblueprintexecs/Responsive-Home-Page-7dedb2131954464fa33664798d0ccc36)


## Implementation description
* 


## Steps to test
1. Check responsiveness using dev tools to verify the homepage looks good on different screens.
2. Give feedback on any improvements required


## What should reviewers focus on?
- Code quality

- While linting I had to make changes in files I knew nothing about. Please check to make sure I didn’t mess up anything. 
       - backend/typescript/rest/authRoutes.ts
       - frontend/src/components/auth/Signup.tsx

NOTE: Sebastian needs to do a thorough self-review before Shrinjay looks at this

## Checklist
